spring-integration-webflux/src/main/java/org/springframework/integration/webflux/dsl/WebFluxMessageHandlerSpec.java

```diff
@@ -144,6 +145,41 @@ public WebFluxMessageHandlerSpec publisherElementTypeExpression(Expression publi
 		return this;
 	}
 
+	/**
+	 * Specify the timeout value for receiving the response form the server.
+	 * If the response is not received with in timeout value, will result in Timeout Exception
+	 * @param timeoutInMillis the timeout in milliseconds
+	 * @return the spec
+	 * @since 5.3
+	 */
+	public WebFluxMessageHandlerSpec timeout(long timeoutInMillis) {
```

Comment:
That's bad name. There are many aspects in Spring Integration components and a lot of different timeout options.
Would be great to be more specific here. How about a `requestTimeout` ?

Also: I wouldn't expose this `long` option. It is not too hard to have that `Duration.ofMillis()` in the target application. 
Plus with Spring Boot configuration properties we can provide an option in the duration style and we'll get exactly `Duration` instance in the POJO